styles/components/_masthead.scss

```diff
@@ -317,23 +317,85 @@
 // Selector Dropdown
 // ==================================================================
 .sprk-c-Masthead__selector-dropdown {
+  background-color: $sprk-white;
   border-bottom: $sprk-masthead-selector-dropdown-border-bottom-mask-open;
-  border-top: 0;
-  border-right: 0;
-  border-left: 0;
-  border-radius: 0;
+  border-radius: $sprk-border-radius;
   box-shadow: $sprk-masthead-selector-dropdown-box-shadow-mask-open;
+  font-size: $sprk-masthead-selector-dropdown-font-size;
+  position: absolute;
   max-width: 100%;
   margin: 0;
```

Comment:
We may want this to be a new $variable.

Reply:
Hey @RVMendoza, looks like you missed this comment from last review.